oecologia-australis.csl

```diff
@@ -268,42 +268,54 @@
           <choose>
             <if match="any" variable="number-of-pages publisher">
               <label variable="number-of-pages" form="short" plural="never"/>
-              <text variable="number-of-pages"/>
+              <text variable="number-of-pages" suffix="."/>
               <text variable="publisher" prefix="ed. " suffix=", "/>
               <text variable="publisher-place"/>
             </if>
           </choose>
         </group>
       </if>
       <else-if type="thesis" match="any">
+        <group suffix=".">
+          <text variable="genre" prefix=" "/>
+        </group>
         <group delimiter=", ">
           <text variable="publisher-place"/>
-          <text variable="publisher"/>
+          <text variable="publisher" prefix=" "/>
         </group>
       </else-if>
       <else-if type="post-weblog webpage" match="none">
-        <group delimiter=", ">
+        <group delimiter=": ">
           <choose>
             <if variable="event version" type="speech" match="none">
               <text variable="genre"/>
             </if>
           </choose>
           <choose>
-            <if type="article-journal article-magazine report" match="none">
-              <group delimiter=", ">
-                <text variable="publisher" font-variant="normal" prefix=" "/>
+            <if type="article-journal article-magazine report legislation" match="none">
+              <group delimiter=": ">
                 <choose>
-                  <if variable="publisher-place">
+                  <if type="legislation"/>
+                  <else-if match="any" variable="publisher-place">
                     <text variable="publisher-place" font-style="normal" prefix=" "/>
-                  </if>
+                  </else-if>
                   <else>
                     <text variable="event-place"/>
                   </else>
                 </choose>
+                <text variable="publisher" font-variant="normal" prefix=" "/>
```

Comment:
this prefix (and the one in line 300) should be moved to the group, too. The space is in the group delimiter.

Reply:
Ok, I got it, but due to an unknown reason the group prefix was not working in some software, I included it here once software excludes dual prefix when two ones work.

Reply:
Ok, I will chang it.